code/modules/clothing/spacesuits/syndi.dm

```diff
@@ -45,11 +47,11 @@
 	icon_state = "syndicate_tan"
 	item_state = "syndi_helm_tan"
 	desc = "A knockoff tan helmet sporting clean lines and durable plating."
+	style = STYLE_NONE
```

Comment:
why would a basic knockoff tan helm have comparable style to the captain's "most fashionable" helm?